src/go/rpk/pkg/cli/cmd/topic_test.go

```diff
@@ -61,19 +61,19 @@ func TestTopicCmd(t *testing.T) {
 			name:           "create should output info about the created topic (custom values)",
 			cmd:            topic.NewCreateCommand,
 			args:           []string{"Seattle", "--partitions", "2", "--replicas", "3", "--compact"},
-			expectedOutput: "Created topic 'Seattle'.\\nYou may check its config with\\n\\nrpk topic describe 'Seattle'\\n\"\n",
+			expectedOutput: "Created topic 'Seattle'.\nYou may check its config with\n\nrpk topic describe 'Seattle'\n\n",
```

Comment:
is this some `go` version stuff or... hard to read

Reply:
Logrus's stderr formatter prints the endline chars, while the stdout one doesn't.